src/block-management/block-toolbar.js

```diff
@@ -1,11 +1,28 @@
+/**
+ * @format
+ * @flow
+ */
+
 import React, { Component } from 'react';
 import { View } from 'react-native';
+import { IconButton } from '@wordpress/components';
 import { BlockFormatControls, BlockControls } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+
+type PropsType = {
+	onInsertClick: void => void,
+};
 
-export default class BlockToolbar extends Component {
+export default class BlockToolbar extends Component<PropsType> {
 	render() {
 		return (
 			<View style={ { height: 50, backgroundColor: '#DCDCDC', flexDirection: 'row' } }>
+				<IconButton
+					className="block-list-appender__toggle"
```

Comment:
Not sure if we need className and what value it has for us?

Reply:
Ah, I left that in to tweak the button's styling later but, I shall remove it from this PR since the styling is not handled here.

Reply:
Done with e8df805.